bitnami/mongodb/templates/replicaset/statefulset.yaml

```diff
@@ -472,30 +394,32 @@ spec:
               /bin/mongodb_exporter --web.listen-address ":{{ .Values.metrics.containerPort }}" --mongodb.uri "{{ include "mongodb.mongodb_exporter.uri" . }}" {{ .Values.metrics.extraFlags }}
           {{- end }}
           env:
-          {{- if .Values.auth.enabled }}
-          {{- if not .Values.metrics.username }}
-          - name: MONGODB_ROOT_USER
-            value: {{ .Values.auth.rootUser | quote }}
-          - name: MONGODB_ROOT_PASSWORD
-            valueFrom:
-              secretKeyRef:
-                name: {{ include "mongodb.secretName" . }}
-                key: mongodb-root-password
-          {{- else }}
-          - name: MONGODB_METRICS_USERNAME
-            value: {{ .Values.metrics.username | quote }}
-          - name: MONGODB_METRICS_PASSWORD
-            valueFrom:
-              secretKeyRef:
-                name: {{ include "mongodb.secretName" . }}
-                key: mongodb-metrics-password
-          {{- end }}
-          {{- end }}
+            - name: BITNAMI_DEBUG
+              value: {{ ternary "true" "false" (or .Values.metrics.image.debug .Values.diagnosticMode.enabled) | quote }}
```

Comment:
Metrics command does not support `BITNAMI_DEBUG` either.

Reply:
You right!